libs/sdk/eslint-config/src/schematics/ng-add/ng-add.schematic.spec.ts

```diff
@@ -13,6 +13,12 @@ import { readJsonFile } from '../shared/utility/tree';
 const COLLECTION_PATH = path.resolve(__dirname, '../../../collection.json');
 const ESLINT_CONFIG_PATH = './.eslintrc.json';
 
+jest.mock('../shared/utility/get-latest-version', () => ({
```

Comment:
What if we needed to add state to the `get-latest-version` script and mock it differently for each test?